src/Components/PresentationalComponents/CSAwRuleBox/CSAwRuleBox.js

```diff
@@ -46,131 +47,130 @@ const CSAwRuleBox = ({ rules, synopsis, changeExposedSystemsParameters, intl })
         dispatch(changeExposedSystemsParameters({ rule: ruleId }));
     };
 
-    return <Fragment>
-        {sortedRules && sortedRules.map((rule, index) => (
-            rule.summary && (
-                <Card className="card-box" key={rule.rule_id} ouiaId={'security-rule-card-' + index}>
-                    <ExpandableSection toggleText={
-                        <Split>
-                            <SplitItem className="pf-u-mr-xl">
+    return sortedRules && sortedRules.map((rule, index) => (
```

Comment:
the rule seems useful but I tend to like more the following code (in terms of code readability) 

```
return ( 
   sortedRules && sortedRules.map((rule, index) => ....
)
```

compared to 

```
return sortedRules && sortedRules.map((rule, index) => ....
```